forge-gui/res/cardsfolder/upcoming/press_the_enemy.txt

```diff
@@ -0,0 +1,9 @@
+Name:Press the Enemy
+ManaCost:2 U U
+Types:Instant
+A:SP$ ChangeZone | ValidTgts$ Permanent.nonLand,Card.inZoneStack | TgtZone$ Stack,Battlefield | Origin$ Battlefield,Stack | Fizzle$ True | Destination$ Hand | RememberChanged$ True | SubAbility$ DBMayPlay | SpellDescription$ Return target spell or nonland permanent an opponent controls to its owner's hand.

```

Comment:
Use `RememberLKI`

Reply:
The second part of the card works only when you return a nonland permanent on the board not on the stack

Reply:
Try RememberChanged$ + `CardManaCostLKI`

Test with X spell as target, it should use the stack cmc

Reply:
So I cast fireball for 6 and countered it with press the enemy but was not allowed to cast Time Warp (5 cmc) in my hand. Is this correct ?

Reply:
not really I'll debug